cds_migrator_kit/rdm/migration/transform/transform.py

```diff
@@ -134,7 +141,10 @@ def _communities(self, json_entry):
         return json_entry.get("communities", [])
 
     def _owner(self, json_entry):
-        email = json_entry["submitter"]
+        email = json_entry.get("submitter")
+        if not email:
+            return "-1"
```

Comment:
@zubeydecivelek Apply same solution on the videos migration please: @jrcastro2

Reply:
we will discuss this on the brainstorming on Monday, I think we should consider what are the consequences. It is also needed for the harvester